docker-compose.yml

```diff
@@ -87,6 +87,10 @@ services:
     image: semitechnologies/reranker-transformers:cross-encoder-ms-marco-MiniLM-L-6-v2
     ports:
       - "8008:8080"
+  multi2vec-bind:
+    image: semitechnologies/multi2vec-bind:imagebind
+    ports:
+      - "8009:8080"
```

Comment:
I think the port needs to be readjusted to not conflict with the gpt4all PR

Reply:
yes, you're right, good point